src/main/java/com/dungeoncrawler/Javiarenka/drop/ItemDropped.java

```diff
@@ -0,0 +1,47 @@
+package com.dungeoncrawler.Javiarenka.drop;
+
+import java.util.Random;
+
+public class ItemDropped {
+    private ItemType droppedItemType;
+    private int itemsDropped;
+
+    public ItemDropped() {
+        this.droppedItemType = null;
+        this.itemsDropped = 0;
+    }
```

Comment:
Czemu ustawiamy tutaj nulla i 0?

Reply:
0 to inicjalizacja w konstruktorze, coś co zapobiega wyskakiwaniu nullPointerów. Czasem zamiast w konstruktorze 0 wpisuje się w deklaracji zmiennej, ale niektórzy uważają że w konstruktorze jest bardziej elegancko. droppedItemType nie wiem czy nie byłoby dobrze wrzucić jako parametr konstruktora, wtedy konstruowalibyśmy ItemDropped już z parametrem np. ItemDropped item = new ItemDropped(ItemType.GEM);